Makefile

```diff
@@ -87,3 +87,7 @@ vendor:
 .PHONY: sanity
 sanity:
 	$(MAKE) vendor && git diff --stat HEAD --ignore-submodules --exit-code
+
+.PHONY: help
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
```

Comment:
:+1: 

Nice, we'll likely want to expand this so it's not a one-liner and a bit easier to parse when you're inspecting the Makefile, but this is nice to have for now.